buildtools/config.py

```diff
@@ -400,7 +400,10 @@ def Verify_WX_CONFIG(self):
         """
         # if WX_CONFIG hasn't been set to an explicit value then construct one.
         if self.WX_CONFIG is None:
```

Comment:
I would propose to replace the if-else statement with the following:
self.WX_CONFIG = os.environ.get("WX_CONFIG",  "wx-config")